storybook/pages/ProfileDialogViewPage.qml

```diff
@@ -0,0 +1,158 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+
+import Storybook 1.0
+
+import utils 1.0
+import shared.views 1.0
+
+SplitView {
+    id: root
+
+    // globalUtilsInst and mainModuleInst injection/replacement
+    QtObject {
+        Component.onCompleted: {
+            Utils.globalUtilsInst = {
+                getEmojiHashAsJson: function(publicKey) {
+                    return JSON.stringify(["👨🏻‍🍼", "🏃🏿‍♂️", "🌇", "🤶🏿", "🏮","🤷🏻‍♂️", "🤦🏻", "📣", "🤎", "👷🏽", "😺", "🥞", "🔃", "🧝🏽‍♂️"])
+                },
+                getColorId: function(publicKey) { return 0 }, // TODO
+                getCompressedPk: function(publicKey) { return publicKey }
+            }
+            Utils.mainModuleInst = {
+                getContactDetailsAsJson: function(publicKey, getVerificationRequest) // TODO make an editor for this
+                {
+                    return JSON.stringify({ displayName: "Mock User Name",
+                                              optionalName: "OriginalMockUser",
+                                              displayIcon: "", // TODO
+                                              publicKey: publicKey,
+                                              name: "mock-ens-name",
+                                              ensVerified: true,
+                                              alias: "Mock User Triplet",
+                                              lastUpdated: Date.now(),
+                                              lastUpdatedLocally: Date.now(),
+                                              localNickname: "MockNickname",
+                                              thumbnailImage: "", // TODO
+                                              largeImage: "", // TODO
+                                              isContact: true,
+                                              isAdded: true,
+                                              isBlocked: false,
+                                              removed: false,
+                                              requestReceived: true,
+                                              hasAddedUs: true, // same as above
+                                              isSyncing: false,
+                                              removed: false,
+                                              trustStatus: Constants.trustStatus.unknown,
+                                              verificationStatus: Constants.verificationStatus.unverified,
+                                              incomingVerificationStatus: Constants.verificationStatus.unverified,
+                                              socialLinks: "", // TODO
+                                              bio: "Hello from MockMainModule, I am a mock user and this is my bio."
+                                          })
+                }
+            }
+        }
+
+        Component.onDestruction: {
+            Qt.callLater(function () {
+                Utils.globalUtilsInst = {}
```

Comment:
I'm not sure if it's better to leave empty js object or `undefined`, as it was

Reply:
I'll leave it as is for now, it is only a mock after all